src/Cat.java

```diff
@@ -0,0 +1,21 @@
+public class Cat extends Animals {
+    private static final int maxRunDistance = 200;
+
+    public Cat(String name, int age) {
+        super(name, age);
```

Comment:
I don't see any counter for cats, but it was required to implement according to the description.